addon/components/admin-input.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-var set = Ember.set;
-var get = Ember.get;
-var computed = Ember.computed;
+const set = Ember.set;
+const get = Ember.get;
+const computed = Ember.computed;
```

Comment:
I had time finally, but it is not work yet.
In browser the tests ok(without destructuring for assignment), but not with "ember test", like at CI.
I don't know whats happen.